### ISSUE TITLE
border colors for ungrouped items

### ISSUE THREAD

**Issue by williamcodes** [stride-nyc/remote_retro#616]
As a user,
I want ungrouped items to have pretty border colors like grouped items have
So that people stop instinctively trying to group absolutely everything.

__Acceptance Criteria:__

- [ ] ungrouped items have pretty border colors like grouped items have

**Comment by vanderhoop** [comment]
Hmmm... I think that gives all items distinct border color could cause more confusion, as this could lead to cases where users think elements _are_ grouped when in fact they are not. 

There's also the issue of running out of (clearly) distinct colors, as right now we have a palette of ~40 colors, and a submission limit of 75 ideas per retro. 

That said, "people stop instinctively trying to group absolutely everything" is a sensible goal. Maybe we can augment or change the guidance copy, or provide automated guidance midway through grouping (once the popcorn has slowed in its popping, so to speak)?

**Comment by williamcodes** [comment]
I don't think you need to worry about people mistakenly believing that cards are grouped, because ungrouped cards will have differing colors, whereas grouped cards will have matching colors. If you're still worried though, maybe make singleton cards have a bold black border once moved. That way they colorize when grouped but still look sorted when they've been moved but left solo.

If you're worried about running out of colors altogether, I would suggest addressing that as a separate story, and I would recommend adding textures to all grouped cards. If you add ten textures, you 10x the skins available. 

Here are some possible textures:

- left diagonal lines
- right diagonal lines
- crisscrossed diagonal lines
- vertical lines
- horizontal lines
- horizontal and left diagonal lines
- sawtooth
- houndstooth
- tiny spots
- large spots
- sawtooth with tiny spots

Add three levels of shading and you 3x the number of skins again. Add three different kinds of corners and you 3x your skins again.

**Comment by williamcodes** [comment]
Bump on this. I'm currently in a retro where unrelated things were grouped and the PM even expressed anxiety about the "orphans" which didn't have a group. It's making the discussion confusing because half the people are talking about one topic in the group and half are talking about the other topic. They just shouldn't have been grouped together. The UX pushed people to do the wrong thing.

**Comment by vanderhoop** [comment]
I'm no longer actually maintaining this project. Maybe @dvndrsn can chat with the current UX folk at Stride about how to fix. My one callout/concern is the border colors being too similar across too many items, but there's probably a creative solution they can arrive at
